src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -85,7 +85,7 @@ public int MaximumFollowRelLink
 
```

Comment:
@iSazonov, your last commit had 158 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Invoke-RestMethod tests.Invoke-RestMethod returns Correct User-Agent on Linux
```powershell
Expected regular expression '.*\(Linux;.*\) PowerShell\/\d+\.\d+\.\d+.*' to case sensitively match $null, but it did not match.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1985
1985:         $result.Output.headers.'User-Agent' | Should -MatchExactly '.*\(Linux;.*\) PowerShell\/\d+\.\d+\.\d+.*'
```
Invoke-RestMethod tests.Invoke-RestMethod returns headers dictionary
```powershell
Expected '127.0.0.1:8083', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 2005
2005:         $result.Output.headers.Host | Should -Be $Uri.Authority
```
Invoke-RestMethod tests.Validate Invoke-RestMethod -DisableKeepAlive
```powershell
Expected '127.0.0.1:8083', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 2016
2016:         $result.Output.headers.Host | Should -Be $uri.Authority
```
Invoke-RestMethod tests.Validate Invoke-RestMethod -MaximumRedirection
```powershell
Expected regular expression '127.0.0.1:8083' to match $null, but it did not match.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 2027
2027:         $result.Output.headers.Host | Should -Match $uri.Authority
```
Invoke-RestMethod tests.Invoke-RestMethod supports request that returns page containing UTF-8 data.
```powershell
Expected regular expression '???? ???  ??????? ???' to match $null, but it did not match.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 2043
2043:         $Result.Output | Should -Match '???? ???  ??????? ???'
```